### DISCUSSION TITLE
Default values for the `MultiSelect`

### DISCUSSION THREAD

**Discussion by mohamed-ahmed-yousef** [mantinedev/mantine#6713] [Q&A] [answered]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.12.1

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

I'm editing an entity from the backend and need to set default values for a MultiSelect component using an array like `[{ label: '', value: '' }]`, where the label is displayed to the user and the value is the UUID that should be sent to the backend when editing the entity.However, I noticed that MultiSelect expects the default value as a `string[]`.
 How can I handle this situation?

### Self-service

- [X] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment, marked as answer]
```tsx
<MultiSelect
  data={[{ value: '1', label: 'First' }, { value: '2', label: 'Second' }]}
  defaultValue={['1', '2']}
/>
```

**Reply by mohamed-ahmed-yousef** [reply]
Thanks a ton!
